src/platform/common/crypto.ts

```diff
@@ -4,10 +4,10 @@
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-import { createHash } from 'crypto';
 import { injectable } from 'inversify';
 import { traceError } from '../logging';
 import { ICryptoUtils, IHashFormat } from './types';
+import * as hashjs from 'hash.js';
```

Comment:
We don't need a secure hash for this stuff, we just use it to create names of files.

Reply:
Plus crypto is deprecated I think.